tests/postgres_tests/test_aggregates.py

```diff
@@ -110,6 +118,23 @@ def test_string_agg_empty_result(self):
         values = AggregateTestModel.objects.aggregate(stringagg=StringAgg('char_field', delimiter=';'))
         self.assertEqual(values, {'stringagg': ''})
 
+    @skipUnlessDBFeature('has_jsonb_datatype')
+    def test_json_agg(self):
+        """
+        Aggregation for a single value JSON field.
```

Comment:
I think these docstrings don't add much value. The test method names seem clear.